### PR TITLE
doc: add proposal for web-terminal

### PR DESCRIPTION
**Pull request by warjiang**
**What type of PR is this?**
/kind design

**What this PR does / why we need it**:
Through web-terminal we can provie standard environment and useful tools for dashboard end-users, make it easily to experience karmada.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/proposal/web-terminal-for-karmada-dashboard/READMD.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Web-terminal for Karmada Dashboard
+
+| title | authors   | reviewers | approvers | creation-date |
+| --- |-----------| --- | --- | --- |
+| Web-terminal for Karmada Dashboard | @warjiang | @ | @ | 2024-07-08 |
+
+# Web-terminal for Karmada Dashboard
```
[the remark this answers is not in the document]
Comment (warjiang) [reply docs/proposal/web-terminal-for-karmada-dashboard/READMD.md:6]: +1, already rm the duplicated title 😃

**Review by samzong — APPROVED** [review]
great ~

**Comment by warjiang** [comment]
> great ~

It seems that we need an extra `lgtm` label ~

**Pull request merged**